### ISSUE TITLE
motor_tip_tilt_full should have the SvgMultiStateLED extended to indicate motion

### ISSUE THREAD

**Issue by aberges-SLAC** [pcdshub/pcdswidgets#152]
Users have found the arrow of the `MSTA` state indicator to be unintuitive, as they believe it is telling them the direction of motion not the state. 

It perhaps worth extending this block here:
https://github.com/pcdshub/pcdswidgets/blob/ac487e768dcf19365e7d6717fc117ecff4fe8abc/pcdswidgets/motion/common/svg_multi_state_led.py#L227-L230

To update the glyph based on the intended motion. You would need to figure out if you're the vertical or horizontal axis, and whether or not you have been inverted, to update the glyph accordingly as a state.

`self._state_dict` and `self._icon_paths` would have to update accordingly (or have methods exposed to append/edit them, despite their names implying privacy).

**Comment by ZLLentz** [comment]
I guess the other option would be to use something other than a directional arrow, like an animated rotating wheel or something like that

**Comment by aberges-SLAC** [comment]
I have a very very very bad feeling about `PyDMSymbol` pointing to a gif, something tells me it will fall apart

**Comment by aberges-SLAC** [comment]
Ah, fears confirmed:
https://slaclab.github.io/pydm/_modules/pydm/widgets/symbol.html#PyDMSymbol.setImageFiles

`QPixmap` will just give you the very first frame everytime. We'd have to `QMovie` it.

**Comment by ZLLentz** [comment]
TIL QMovie exists

**Comment by moyinoluwa-10** [comment]
Fixed in #153
